browser/src/robocorp/browser/__init__.py

```diff
@@ -25,49 +25,39 @@
 
 def configure(**kwargs) -> None:
     """
-    May be called before any other method to configure the browser settings.
-
-    Calling this method is optional (if not called a default configuration will
-    be used -- note that calling this method after the browser is already
-    initialized will have no effect).
+    Configures browser settings before any other method is called.
 
+    This method is optional, and if not invoked, default configurations will be used.
+    Note that calling this method after the browser is initialized has no effect.
+    
+    Example:
+        ```python
+        browser.configure(browser_engine="firefox", slowmo=100)
+        ```
+    
     Args:
-        browser_engine:
-            Browser engine which should be used
```

Comment:
Right now lazydocs doesn't turn this into bullet point if the description of the argument begins with new line. If we don't like it with line continuation as a standard we could look into lazydocs and change the behaviour there.

Reply:
@OvidiuCode what do you think would provide the best output format here?

Reply:
I think this is very readable. I like it a lot with newline so I adjusted to code to work with newline.

Reply:
Great.